src/chrome/content/rules/NetEase.xml

```diff
@@ -0,0 +1,18 @@
+<ruleset name="NetEase">
+
+	<target host="nos.netease.com" />
```

Comment:
I can find a lot of other hosts here and please always cover/document ^netease.com/www.netease.com.